src/Select.js

```diff
@@ -174,11 +184,26 @@ Select.STATE = {
 	},
 
 	/**
+	 * Block or unblock the component behaviours.
+	 * @type {?boolean}
+	 * @default false
+	 */
+	disabled: {
+		validator: core.isBoolean,
+		value: false
+	},
+
+	/**
 	 * Flag indicating if the select dropdown is currently expanded.
 	 * @type {boolean}
 	 */
 	expanded_: {
-		validator: core.isBoolean,
+		setter: function(value) {
+			return !this.disabled ? value : false;
+		},
+		validator: function() {
+			return core.isBoolean;
```

Comment:
This should either be `return core.isBoolean()` or just keep the old syntax of `validator: core.isBoolean` (which is cleaner, btw :) )